queries/places.jinja2

```diff
@@ -38,7 +38,7 @@ SELECT
     NULL AS labelrank,
     tags->'population' AS population,
     tags->'capital' AS capital,
```

Comment:
Can you set this as `country_capital`, and looks like Ottowa is getting `capital:yes` exported now, instead of `true`.